perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -2211,6 +2239,24 @@ bool MapBasedPredictionNode::isDuplicated(
 
   return false;
 }
+
+std::optional<lanelet::Id> MapBasedPredictionNode::getTrafficSignalId(
+  const lanelet::ConstLanelet & way_lanelet)
+{
+  const auto traffic_light_reg_elems =
+    way_lanelet.regulatoryElementsAs<const lanelet::TrafficLight>();
+  if (traffic_light_reg_elems.empty()) {
+    return std::nullopt;
+  } else if (traffic_light_reg_elems.size() > 1) {
+    RCLCPP_ERROR(
+      get_logger(),
+      "[Map Based Prediction]: "
+      "Multiple regulatory elements as TrafficLight are defined to one lanelet object.");
+    return std::nullopt;  // Is it appropriate?
```

Comment:
@soblin 
Could you check this function?
What should we return in this line?

Reply:
I heard this type of inappropriate map configuration is checked outside of the autoware.
Hence I make to this line return the front element since it is trivial.